css/styles.css

```diff
@@ -0,0 +1,177 @@
+html, div, ul, ol {
+  margin: 0; }
+
+body {
+  margin: 0;
+  background-image: url("../res/backgrounds/mkbg8.jpg");
+  font-size: 1em;
+  overflow: hidden; }
+
+ul {
+  background-color: black; }
+
+li {
+  display: inline-block; }
+
+img {
+  height: 100%;
+  width: 100%; }
+
+input {
+  height: 30px;
+  width: 90%;
+  font-size: 1em;
+  padding: 0 5px;
+  color: yellow;
+  background-color: black; }
+
+select {
+  height: 30px;
+  width: 90%;
+  font-size: 1em;
+  padding: 0 5px;
+  color: yellow;
+  background-color: black; }
+
+h1 {
+  font-size: 2em; }
+
+h2 {
+  font-size: 1.25em; }
+
+hp {
+  font-size: 1.35em; }
+
+pop {
+  color: #ff5555;
+  font-size: 1.25em;
+  font-weight: 700;
+  margin: 5px 0; }
+
+popGreen {
+  color: #55bb55;
+  font-size: 1.25em;
+  font-weight: 700;
+  margin: 5px 0; }
+
+#battleLog {
+  height: 250px;
+  background-color: black;
+  padding: 10px 20px;
+  text-align: center; }
+
+#pImageHolder {
+  height: 300px; }
+
+#btnClose {
+  cursor: pointer;
+  text-align: center; }
+
+#playerSetup {
+  height: 500px;
+  margin: 0 auto;
+  top: 60px; }
+
+#pressStart {
+  cursor: pointer;
+  height: 50px; }
+
+#startFooter {
+  margin: 0 auto; }
+
+#attackBtn {
+  padding: 0;
+  cursor: pointer;
+  height: 100%; }
+
+#attack {
+  padding: 0;
+  height: 100%; }
+
+#chooseCharacter {
+  padding: 0;
+  background-color: black;
+  height: 100%;
+  max-height: 500px;
+  max-width: 600px; }
```

Comment:
Generally, it's better to avoid styling ids. They are harder to override later if you need to do so, since adding a class to an element dynamically can't change styling added via id. If you know for sure an element will never be changed, then an id is OK, but I think it's best to avoid styling ids when you can.